readme.md

```diff
@@ -33,7 +33,107 @@ npm install colette --save
 
 ---
 ## Download
-- See the [`dist/`](https://github.com/20minutes/colette/dist/) 
+- See the [`dist/`](https://github.com/20minutes/colette/dist/)
 directory for ready-to-use css files.
-- If you want to use Colette as a kickstarter for custom stylus/css 
+- If you want to use Colette as a kickstarter for custom stylus/css
 projects, check [`src/`](https://github.com/20minutes/colette/assets/).
+
+---
+## Use gulp
+
+To compile your stylus, use `gulp` and `gulp-stylus`
+
+```javascript
+/* gulpfile.js */
+gulp.task('css', function () {
+   return gulp.src(cfg.cssDir + '*.styl')
+   .pipe(stylus({
+        include: [
+            'node_modules',
+            'node_modules/colette/node_modules'
+        ],
+        'include css': true
+    }))
+   .pipe(gulp.dest('css/'))
+})
+```
+
+---
+## Partial import
+
+Colette is a collection of stylus features. You can define which styles you want to include into your final css file with stylus import.
+
+By default, you include all `colette.styl` and all features are in your css.
+You can choo only features needed by copying `colette.styl` content and pecise your imports rules.
+
+```
+/* my_css_file.styl */
+
+// Your own setings
+
+// Import colette
+@require 'colette/assets/styl/colette'
+
+/* => will compile to my_css_file.css */
+```
+
+But can be usefull:
+* to change order of imports,
+* import only needed features,
+* separe your critical css from your global css.
+
+### Handle critical css
```

Comment:
O would mention that this is an example of use, eg `Usage example: Handle critical css`